portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,10 +37,30 @@
   
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String text = getParameter(request, "text-input", "");
-    String[] comments = text.split("\\s*,\\s*");
-    response.setContentType("text/html;");
-    response.getWriter().println(Arrays.toString(comments));
```

Comment:
Are you trying to remove all the  "\\s*,\\s*"? If yes, I guess string.replace() works too?

Reply:
Never mind, this line is removed

Reply:
All good!